lib/components/form/batch-settings.tsx

```diff
@@ -222,15 +228,14 @@ class BatchSettings extends Component<Props, State> {
               id: 'components.BatchSettings.planTripTooltip'
             })}
           >
-            <Icon
-              className="fa-2x"
-              type={
-                hasValidLocation(currentQuery, 'from') ||
-                hasValidLocation(currentQuery, 'to')
-                  ? 'refresh'
-                  : 'search'
-              }
-            />
+            <StyledIconWrapper noMargin size="2x">
+              {hasValidLocation(currentQuery, 'from') ||
+              hasValidLocation(currentQuery, 'to') ? (
+                <Sync />
```

Comment:
This icon is less round than the previous one and I think it looks worse. Are there others to choose from?

Reply:
Switched it to the alt version, which I think is the same as what we had before!